dev/archery/archery/docker/cli.py

```diff
@@ -46,10 +46,19 @@ def _execute(self, *args, **kwargs):
               callback=validate_arrow_sources,
               help="Specify Arrow source directory.")
 @click.option('--dry-run/--execute', default=False,
-              help="Display the docker-compose commands instead of executing "
-                   "them.")
+              help="Display the docker commands instead of executing them.")
+@click.option('--using-docker-cli', default=False, is_flag=True,
```

Comment:
With docker-compose being deprecated, shouldn't we default to use docker cli? That way using the deprecated, potentially outdated tool requires an explicit opt-in.

Reply:
We can probably default at some point, but I don't want to disrupt things too much here.